dev/doc/README.md

```diff
@@ -0,0 +1,90 @@
+# Beginner's guide
+
+## Getting dependencies
+
+Assuming one is running Ubuntu (if not, replace `apt` with the package manager of choice)
+
+```
+$ sudo apt-get install make opam git
+$ opam init --comp 4.02.3
+# Then follow the advice displayed at the end as how to update your
+  ~/.bashrc and ~/.ocamlinit files.
+
+$ source ~/.bashrc
+
+# needed if you want to build "coqtop" target
+$ opam install camlp5
+
+# needed if you want to build "coqide" target
+$ sudo apt-get install liblablgtksourceview2-ocaml-dev \
+  libgtk2.0-dev libgtksourceview2.0-dev
+$ opam install lablgtk
+
+# needed if you want to build "doc" target
+$ sudo apt-get install texlive-latex-recommended \
+  texlive-fonts-extra texlive-math-extra \
+  hevea texlive-latex-extra latex-xcolor
```

Comment:
This info is outdated. It's probably better to just link to `doc/README.md` to get this information, isn't it?

Reply:
Possibly, this was stuff that was in `dev/doc/install.txt` that I simply retained.